# 2.083+4*0.208
mass: 2.915
rotor_num: 4.0
rotor_c: 8.54858e-06

pid_x:
  Kp: 0.65
  Ki: 0.0
  Kd: 0.03
  limit:
    high: 500
    low: -500
pid_vx:
  Kp: 0.11
  Ki: 0.0
  Kd: 0.0
  limit:
    high: 500
    low: -500

pid_y:
  Kp: 0.65
  Ki: 0.0
  Kd: 0.03
  limit:
    high: 500
    low: -500
pid_vy:
  Kp: 0.11
  Ki: 0.0
  Kd: 0.0
  limit:
    high: 500
    low: -500

pid_z:
  Kp: 100.0
  Ki: 10.0
  Kd: 100.0
  limit:
    high: 500
    low: -500
pid_vz:
  Kp: 1.0
  Ki: 0.0
  Kd: 0.0
  limit:
    high: 500
    low: -500